lib/sig0.js

```diff
@@ -97,6 +97,18 @@ sig0.createKey = function createKey(algorithm, publicKey) {
   rd.flags = 0;
   rd.protocol = 0;
   rd.algorithm = algorithm;
+
+  // https://tools.ietf.org/html/rfc4034#appendix-A.1.1
+  // PRIVATEDNS publicKey should be wire encoded and prefixed with algo name
+  if (algorithm === sig0.algs.PRIVATEDNS) {
+    const name = "secp256k1sha256";
```

Comment:
Maybe this is the 3rd argument to the function? It would allow for this to be generalized to any `PRIVATEDNS` implementation, which is something that we may have to build on more in the future.

The name `SECP256K1SHA256` could live in the constants file someplace

Reply:
Yes, my initial commit added a third arg but there are multiple callers to this func (`sign` `verify) which don't pass it... needs some more thought but definitely needs to be flexible to allow other names.

Reply:
Hmm maybe this should be a (static?) class implementing an interface which encapsulates the prefix, signer and versifier functions?

Reply:
Leaving this for now. I think this would be good to have but not a blocker atm.